assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/useCellIcon.tsx

```diff
@@ -20,9 +16,9 @@ const useCellIcon = (): Callback => {
 			switch (status) {
 				case 'NEW':
 				case 'OLD':
-					return () => <AntIcon style={iconStyle} component={Ticket} />;
+					return () => <Ticket fill='white' />;
 				case 'REMOVED':
-					return () => <CloseOutlined style={iconStyle} />;
+					return () => <CloseOutlined fill='white' />;
 				default:
 					return () => <MinusOutlined />;
```

Comment:
should add `fill='white'` to the `MinusOutlined` as well

Reply:
it wasn't added for some reasons before and I thought it was intended
***
with white fill feels better:

![image](https://user-images.githubusercontent.com/1477453/80234415-cea31c80-8660-11ea-85d9-0d1b9e4a0489.png)